view/frontend/web/js/view/delivery-options.js

```diff
@@ -152,10 +162,10 @@ define(
        * Create the div the delivery options will be rendered in, if it doesn't exist yet.
        */
       render: function() {
-        var hasUnrenderedDiv = document.querySelector('#myparcel-delivery-options');
-        var hasRenderedDeliveryOptions = document.querySelector('.myparcel-delivery-options__table');
-        var shippingMethodDiv = document.getElementById('checkout-shipping-method-load');
-        var deliveryOptionsDiv = document.createElement('div');
+        const hasUnrenderedDiv = document.querySelector('#myparcel-delivery-options');
```

Comment:
Dit moet je eigenlijk niet vervangen wegens browser compatibility, ie11 support dit niet helemaal, en lagere browsers helemaal niet. Dit kan je beter terugdraaien tot we een fatsoenlijke js build flow hebben geimplementeerd